python/chronos/example/onnx/README.md

```diff
@@ -5,10 +5,11 @@ In this example, onnx speed up the inferencing for ~4X.
 ## Prepare the environment
 We recommend you to use Anaconda to prepare the environment, especially if you want to run on a yarn cluster:
 ```bash
-conda create -n zoo python=3.7 # "zoo" is conda environment name, you can use any name you like.
-conda activate zoo
-pip install --pre --upgrade analytics-zoo[automl]
+conda create -n my_env python=3.7 # "my_env" is conda environment name, you can use any name you like.
+conda activate my_env
+pip install bigdl-chronos
 ```
+For more details, please refer to [here](https://bigdl.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html).
```

Comment:
We could remove the commands and keep the below words only.
`You can refer to [Chronos Install Guide]()`